samples/Calling/src/app/App.tsx

```diff
@@ -191,18 +191,20 @@ const App = (): JSX.Element => {
         return <Spinner label={'Getting user credentials from server'} ariaLive="assertive" labelPosition="top" />;
       }
       return (
-        <CallScreen
-          token={token}
-          userId={userId}
-          displayName={displayName}
-          callLocator={callLocator}
-          /* @conditional-compile-remove(PSTN-calls) */
-          alternateCallerId={alternateCallerId}
-          /* @conditional-compile-remove(rooms) */
-          roleHint={role}
-          /* @conditional-compile-remove(teams-identity-support) */
-          isTeamsIdentityCall={isTeamsCall}
-        />
+        <React.StrictMode>
```

Comment:
I guess this StrictMode element was added for testing purposes? Remove it if there is no specific reason to add this

Reply:
yup this was added so that should this come up again on the calling side we will know.